src/lib.rs

```diff
@@ -317,6 +317,26 @@ impl Check {
             .deps(false)
             .silence(!config.is_verbose());
 
+        // If both the current and baseline rustdoc are given explicitely as a file path, we don't need to use the installed rustc, and this check can be skipped.
+        if !(matches!(self.current.source, RustdocSource::Rustdoc(_))
+            && matches!(self.baseline.source, RustdocSource::Rustdoc(_)))
+        {
+            match rustc_version::version() {
+                Ok(rustc_version) => {
+                    let rustc_version_needed = rustc_version::Version::new(1, 65, 0);
+                    if rustc_version < rustc_version_needed {
+                        anyhow::bail!("rustc version is not high enough: >={} needed, got {}\n\nHELP: to run with the latest rustc version, use `rustup update stable && cargo +stable semver-checks ...`", rustc_version_needed, rustc_version);
```

Comment:
```suggestion
                        anyhow::bail!("rustc version is not high enough: >={} needed, got {}\n\nHELP: to run with the latest rustc version, use `rustup update stable && cargo +stable semver-checks <arguments-here>`", rustc_version_needed, rustc_version);
```
Just out of concern someone might copy-paste this command without noticing that the `...` isn't an actual argument to `cargo-semver-checks`.